### PR TITLE
Switch iconset to remix icons

### LINKED ISSUE #1348: Unify icon naming
Currently, icons are named pretty inconsistently. Would be cool to have a naming convention on whether how to proceed with e.g. variations and if dashes or underscores in their names should be used.

Note that this would perhaps bring another breaking change

### PR DESCRIPTION
**Pull request by lookacat**
## Description
Switch from our rather random collection of icons to Remix icons

## Related Issue
- Fixes #1348

## Types of changes
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation added/updated

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by pascalwengerter — COMMENTED** [review]
I know this is in a draft state, first feedback
- needs a `change` changelog item
- attribution/reference to Remix icons on https://github.com/owncloud/owncloud-design-system/blob/master/docs/components/tokens/Icon.vue

**Comment by pascalwengerter** [comment]
This also blows the `dist/system/` from 870 KB to 2.6 MB, we'll have to discuss if that's worth it for us or if we can drop some of the more uncommon icons from the Remix set

**Review by pascalwengerter — COMMENTED** [review]
Please make sure this doesn't overwrite https://github.com/owncloud/owncloud-design-system/pull/1846 or provides a similiar solution

**Review by pascalwengerter — COMMENTED** [review]
Probably known but the current state of this branch has a broken icon page (http://localhost:6060/#/Design%20Tokens/Icon)

File: package.json [hunk 1]
```diff
@@ -153,5 +153,8 @@
   "engines": {
     "node": ">= 14.0.0",
     "npm": ">= 3.0.0"
+  },
+  "dependencies": {
+    "cpx": "^1.5.0"
```
Comment (pascalwengerter) [inline package.json:158]: What's this being used for  @lookacat ?:)

**Comment by pascalwengerter** [comment]
> Probably known but the current state of this branch has a broken icon page (http://localhost:6060/#/Design%20Tokens/Icon)

Has been resolved, changelog item added. Last two things are checking if some sort of "Spaces" icon is there and getting approval from @kulmann so we can go ahead in web 🥳

**Comment by pascalwengerter** [comment]
> Probably known but the current state of this branch has a broken icon page (http://localhost:6060/#/Design%20Tokens/Icon)

Just checked and it's still there:)

**Review by kulmann — APPROVED** [review]
Please squash on merge

**Pull request merged**